pythran/pythonic/types/numpy_texpr.hpp

```diff
@@ -200,5 +200,15 @@ struct __combined<pythonic::types::numpy_texpr<E>, K> {
     typedef pythonic::types::numpy_texpr<E> type;
 };
 
+template<class E, class T, size_t N>
+struct __combined<pythonic::types::numpy_texpr<E>, pythonic::types::ndarray<T, N>> {
+    typedef pythonic::types::ndarray<T, N> type;
+};
+
+template<class E, class T, size_t N>
+struct __combined<pythonic::types::ndarray<T, N>, pythonic::types::numpy_texpr<E>> {
+    typedef pythonic::types::ndarray<T, N> type;
+};
```

Comment:
Hum ok, aliasing issue.
We are really not good enough on these kind of issue as even for  the Pythran compiler (compile time part) do not detect this AFAIK.